src/components/network-chart/NetworkChartRow.js

```diff
@@ -116,6 +129,63 @@ class NetworkChartRowBar extends React.PureComponent<NetworkChartRowBarProps> {
     return markerPosition;
   }
 
+  _preconnectComponent(): React.Node {
```

Comment:
To tell the truth, I couldn't make a profile where this preconnect component would be visible, so I can't give you an example of this. But in theory this should be possible and could happen.